pubtools/_quay/operator_pusher.py

```diff
@@ -506,12 +520,14 @@ def build_index_images(self):
                 )
                 failed_items[id(item)] = True
 
+        # if any of push items failed due to fbc issues, return early and skip all iib operations
+        if failed_items:
+            return {}
```

Comment:
Don't we want to raise instead?

Reply:
That will be handled in push_docker. It already does check if items contain errors or not